cardano-cli/src/Cardano/CLI/EraBased/Options/Common.hs

```diff
@@ -2231,15 +2244,15 @@ pTxInCount =
   fmap TxInCount $ Opt.option Opt.auto $ mconcat
     [ Opt.long "tx-in-count"
     , Opt.metavar "NATURAL"
-    , Opt.help "The number of transaction inputs."
+    , Opt.help "DEPRECATED. This argument has no effect."
     ]
 
 pTxOutCount :: Parser TxOutCount
 pTxOutCount =
   fmap TxOutCount $ Opt.option Opt.auto $ mconcat
     [ Opt.long "tx-out-count"
     , Opt.metavar "NATURAL"
-    , Opt.help "The number of transaction outputs."
+    , Opt.help "DEPRECATED. This argument has no effect."
```

Comment:
Can you rename the enclosing function to `pTxOutCountDeprecated`? Otherwise new callers could be tempted to use this function, where as they should not.

Reply:
Done!